x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_table/cell_renderer.tsx

```diff
@@ -63,6 +63,6 @@ export const cellRendererFactory = (from: number) => {
       return displayValue(indicator) || EMPTY_VALUE;
     }
 
-    return unwrapValue(indicator, columnId as RawIndicatorFieldId) || EMPTY_VALUE;
+    return IndicatorField({ indicator, field: columnId, fieldTypesMap });
```

Comment:
and here is well, there is a difference between a Component and running a function that returns jsx

Reply:
awesome thanks for the link, reading it now!

Reply:
done